### PR TITLE
Added use of the new data model fetching API

### PR DESCRIPTION
**Pull request by mikeger**
Integrated the new API from Data Model.

### CODE REVIEW

**Comment by mikeger** [comment]
PR updated with links support and autolayout for cells

File: Wire-iOS/Sources/UserInterface/Collections/CollectionFileCell.swift [hunk 1]
```diff
@@ -47,6 +47,24 @@ final public class CollectionFileCell: UICollectionViewCell, Reusable {
         self.loadView()
     }
     
+    var isHeightCalculated: Bool = false
+    var containerWidth: CGFloat = 320
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Collections/CollectionFileCell.swift]: Should be hardcoded?

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Collections/CollectionFileCell.swift]: Just the default value, actually set when the cell is created.

**Comment by mikeger** [comment]
@typfel updated the PR

**Pull request merged**